### ISSUE TITLE
Accessibility (WCAG level A) Skipped header levels on Statistics landing page.

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1182]
1.  Go to https://nuget.org/stats and view source
2.  Scrutinize how we're using h1..h6 tags to organize headers.

Result:  On this page, there are h3 elements, but not any h2 or h1 elements.
Expected:  We shouldn't skip levels when using the h1..h6 tags. In other words, if we're using an h3 tag, that means there should be an h1 and h2 tag somewhere else on the page. In this case, I'd expect us to use h1 tags and re-style the tags as necessary to achieve the visual effect we want.
